.github/workflows/ci.yml

```diff
@@ -61,4 +61,38 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Push snapshot container image
-        run: docker push ghcr.io/caas-team/sparrow:${{ steps.version.outputs.value }}
\ No newline at end of file
+        run: docker push ghcr.io/caas-team/sparrow:${{ steps.version.outputs.value }}
+  helm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+
+      - name: Fetch tags explicitly
+        run: git fetch --tags
```

Comment:
You're probably doing this because of the issue right? 
https://github.com/actions/checkout/issues/1471

Reply:
yep